File: app/src/main/java/com/zhangw/aliencat/video/videoplayer/PlayState.java
package com.zhangw.aliencat.video.videoplayer;

/**
 * 视频播放状态
 * Created by xiemy on 2017/6/7.
 */
public class PlayState {
    //预备
    public final static int READY = 0xF1;
    //加载
    public final static int LOADING = 0xF2;
    //播放
    public final static int PLAY = 0xF3;
    //暂停
    public final static int PAUSE = 0XF4;
    //停止
    public final static int STOP = 0xF5;
    //播放完成
    public final static int COMPLETE = 0Xf6;
    //释放
    public final static int RELEASE = 0xF7;
}

File: app/src/main/java/com/zhangw/aliencat/video/videoplayer/PlayState.kt
package com.zhangw.aliencat.video.videoplayer

/**
 * 视频播放状态
 * Created by xiemy on 2017/6/7.
 */
object PlayState {
    //正常
    const val STATE_NORMAL = 0xF0
    //预备中
    const val STATE_PREPAREING = 0xF1
    //播放中
    const val STATE_PLAYING = 0xF2
    //开始缓存
    const val STATE_BUFFERING_START=0xF3
    //暂停
    const val STATE_PAUSE = 0XF4
    //自动播放结束
    const val STATE_AUTO_COMPLETE = 0xF5
    //错误状态
    const val STATE_ERROR = 0XF6
}
